src/layout/navigation/Sidebar.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { NavLink, Link } from 'react-router-dom';
+import './Navigation.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faYoutube,
+  faFacebook,
+  faTwitter,
+  faInstagram,
+  faGooglePlus,
+} from '@fortawesome/free-brands-svg-icons';
+
+const Navigation = () => (
+  <nav className="sidenav">
+    <div className="d-flex flex-column justify-content-between" style={{ height: '50%' }}>
+      <div>
+        <Link to="/">
+          <h1 className="text-center text-dark"><i>Power Motors</i></h1>
+        </Link>
+      </div>
+      <div className="d-flex flex-column center-align">
+        <NavLink to="#" exact activeClassName="active" className="nav-link p-3">MOTORSCYCLES</NavLink>
+        <NavLink to="#" activeClassName="active" className="nav-link p-3">RESERVATIONS</NavLink>
+        <NavLink to="#" activeClassName="active" className="nav-link p-3">MY RESERVATION</NavLink>
+        <NavLink to="#" activeClassName="active" className="nav-link p-3">ADD MOTORCYCLE</NavLink>
```

Comment:
Links on the nav.

- [ ] Motorcycles
- [ ]  "Reserve" form.
- [ ]  My reservations.
- [ ]  Add motorcycle
- [ ]  Delete motorcycle

Please make sure your navigation links have the above names.